CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing to OSRD
+
+## Where to start ?
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+If you are brand new to OSRD or open-source development, we recommend chatting with us on [libera chat](https://web.libera.chat/#osrd) or going through the [GitHub issues](https://github.com/DGEXSolutions/osrd/issues) tab to find issues that interest you. There are number of issues listed under `good first issue` where you could start out.
+
+## Contribute code
+
+Commit style is detailed in the [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code/#git-commit-style).
+
+### Pull request / code review
```

Comment:
In all this paragraph, it seems to me that there is no logical order between each point. I suggest to reorder them chronologically:
- keep your branch up to date
- make atomic commits
- add a description
- assign a reviewer
- answer to the reviewer before asking for another review
- etc